routes/api_worldometers.js

```diff
@@ -38,8 +38,10 @@ router.get('/countries', async (req, res) => {
 router.get('/countries/:query', async (req, res) => {
 	const { query } = req.params;
 	const countries = JSON.parse(await redis.get(keys.countries)).map(fixApostrophe);
+	const isNotStrict = req.query.strict === 'false';
+	const isStrict = !isNotStrict;
```

Comment:
please change to
`const isStrict = !(req.query.strict === 'false')`

Reply:
done. also for all endpoints using strict queryparam.

Reply:
my bad, change to `const isStrict = req.query.strict !== 'false'`